testing.md

```diff
@@ -0,0 +1,37 @@
+# Manual Testing (for Native Query)
+
+At this time, Native Query must be tested manually by reviewers.
+
+The following query is sufficient to test Native Query as it exercises mutliple areas of translation.
+
+1. Ensure you have a local ADF running.
+- Navigate to the `mongo-odbc-driver` repository locally.
+- Run the ADF script via `./resources/run_adf start`
+- After it sets up and returns back to the shell prompt, load local data via `cargo run --bin data_loader`
+- Navigate back to the `mongo-powerbi-connector` repository.
+
+2. Build the connector. The easiest way to do this is in VSCode, using the Power Query SDk.
+- Navigate to the `connector` directory.
+- `code .` to open the project in VSCode.
+- Open `MongoDBAtlasODC.pq`
+- Right click, and choose *Evaluate current power query file*
+- Copy `bin\AnyCPU\Debug\connector.mez` to the Microsoft Power BI custom connector folder, located in `C:\Users\<username>\Documents\Microsoft Power BI Desktop\Custom Connectors`.
```

Comment:
```suggestion
- Copy `bin\AnyCPU\Debug\connector.mez` to the Microsoft Power BI custom connector folder, located in `C:\Users\<username>\Documents\Power BI Desktop\Custom Connectors`.
```


Is your Power BI version the custom one?  My custom connector goes into: `C:\Users\[username]\Documents\Power BI Desktop\Custom Connectors`

> Put the connector .pq, .pqx, .m, or .mez file in your local [Documents]\Power BI Desktop\Custom Connectors folder. If the folder doesn't exist, create it.
https://learn.microsoft.com/en-us/power-bi/connect-data/desktop-connector-extensibility#custom-connectors

Reply:
This is how I've always had it. I wonder if they allow both, as I seem to remember their docs specifically called out the name as `Microsoft Power BI Desktop`. I have both the versions you download from the internet or that you can install via the windows store.

I searched some on this and it appears that it could be named both, but don't have both. https://community.fabric.microsoft.com/t5/Developer/Custom-Connectors-Path/m-p/706338#M19564

I can update this if you want.

Reply:
Oh, Yeah let's update it since their official doc says `Power BI Desktop`.